### ISSUE TITLE
Correction to Anthology ID 2020.wmt-1.73

### ISSUE THREAD

**Issue by tomsbergmanis** [acl-org/acl-anthology#1195]
## Metadata correction: please describe the issue here**
Author order in metadata/bibtex is different than that in the paper. 
It should be: 
author = "Stafanovi{\v{c}}s, Art{\=u}rs  and Bergmanis, Toms and Pinnis, M{\=a}rcis",

Corrected bib in text format added below. 
[2020.wmt-1.73.bib.txt](https://github.com/acl-org/acl-anthology/files/5917421/2020.wmt-1.73.bib.txt)

Thanks! 
Toms

**Comment by mjpost** [comment]
Hi @tomsbergmanis—[Anthology policy](https://www.aclweb.org/anthology/info/corrections/#note-on-author-changes) requires you to get permission from the WMT organizers. Please ask them to comment here or to email me privately. We should also hear from your coauthors.

**Comment by tomsbergmanis** [comment]
I can ask my co-authors, I can't imagine a WMT organiser who would care, but sure.

**Comment by artursstaf** [comment]
Aye

**Comment by mjpost** [comment]
Yes, permission for these requests is usually pro-forma. However, it is the policy.

**Comment by pmarcis** [comment]
I agree to the correction!

**Comment by bhaddow** [comment]
I agree to the changes
